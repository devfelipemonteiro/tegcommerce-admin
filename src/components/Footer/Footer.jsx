import React from 'react';
import { pure } from 'recompose';
import Style from './style/Footer.scss';

const Footer = () => (
  <div className={Style.root}>
    <div className={Style.container}>
      <div className={Style.footerLeftContainer}>Cicero Viana</div>
      <div className={Style.footerRightContainer}>github.com/cicerohen</div>
    </div>
  </div>
);

export default pure(Footer);
