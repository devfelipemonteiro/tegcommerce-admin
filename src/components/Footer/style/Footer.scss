@import "./Footer.variables";

.root {
  background-color: $footerBgColor;
  color: $footerColor;
}

.container {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding-left: $default-padding;
  padding-right: $default-padding;
  height: $footerHeight;
}

.footerLeftContainer {
  display: flex;
  flex: 1 1 auto;
  justify-content: flex-start;
}

.footerRightContainer {
  display: flex;
  flex: 1 1 auto;
  justify-content: flex-end;
}