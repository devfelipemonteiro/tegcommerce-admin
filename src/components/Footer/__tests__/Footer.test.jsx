import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

describe('<Footer /> Tests', () => {
  it('should render without exploding', done => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toHaveLength(1);
    done();
  });

  it('should render correctly', done => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
    done();
  });
});
