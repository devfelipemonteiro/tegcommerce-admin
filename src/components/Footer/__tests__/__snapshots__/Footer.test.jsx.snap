// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Tests should render correctly 1`] = `<Footer />`;
